test/integration/data-types/data-types.test.ts

```diff
@@ -215,7 +215,7 @@ describe('DataTypes', () => {
       expect(pick(user.get(), ['tinyText', 'mediumText', 'longText'])).to.deep.eq(data);
     });
 
-    // TODO: once we have centralized logging, check a warning message has been emitted when length is not supported:
+    // TODO!: once we have centralized logging, check a warning message has been emitted when length is not supported:
```

Comment:
Here

Reply:
I've regarded this a fine one for a future PR, the others have been unresolved in the main PR